locations/spiders/tsb_gb.py

```diff
@@ -14,3 +14,8 @@ class TSBGB(SitemapSpider, StructuredDataSpider):
     sitemap_urls = ["https://branches.tsb.co.uk/sitemap.xml"]
     sitemap_rules = [(r"https:\/\/branches\.tsb\.co\.uk\/[-\w]+\/[-\/\w]+\.html$", "parse_sd")]
     wanted_types = ["BankOrCreditUnion", "FinancialService"]
+
+    def post_process_item(self, item, response, ld_data, **kwargs):
+        if item["image"] == item["website"]:
+            item["image"] = None
+        yield from self.inspect_item(item, response)
```

Comment:
We don't really do `inspect_item` any more.

```suggestion
        yield item
```